src/views/Profile.js

```diff
@@ -84,9 +84,11 @@ class Profile extends React.Component {
             <FormFieldValidator invalid={!!this.state.requestFailedMessage} message={this.state.requestFailedMessage} />
           </FormFieldAlertRow>
           <br />
-          <FormFieldWideRow className='text-center'>
-            <Button variant='primary' onClick={this.handleShowModal}>Edit Details</Button>
-          </FormFieldWideRow>
+          <div className='row'>
+            <div className='col-md-12 text-center'>
+              <Link to='/EditDetails'><button className='btn btn-primary'>Edit Details</button></Link>
+            </div>
+          </div>
```

Comment:
Keep the parent tag, and it works just like any HTML tag, as far as children tags are concerned. Also, we should prefer the `react-bootstrap` `<Button>` tag to the HTML `<button>` tag, wherever we actually want Bootstrap classes like `btn` and `btn-primary`.
```
<FormFieldWideRow className='text-center'>
    <Link to='/EditDetails'><Button variant='primary'>Edit Details</Button></Link>
</FormFieldWideRow>
```

Reply:
Done.

Reply:
I understand that you just resolved these with the intent to do them, but you missed this one. I can approve once this one is also enacted.